themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -499,8 +855,8 @@ expect(await page.title()).toBe("Pulumi Challenge");
 await browser.close();
 ```
 
-Congratulations! You completed the first Pulumi Challenge. If you’d like to tear down all of these resources, run `pulumi destroy`. Otherwise, enjoy the new website!
+Congratulations! You completed the first Pulumi Challenge. If you'd like to tear down all of these resources, run `pulumi destroy`. Otherwise, enjoy the new website!
```

Comment:
Ah, I see -- it comes up "eventually". 

We should totally mention to expect a lag here, so people aren't left scratching their heads.

Reply:
Fixed in 1e5040bd. I added text to instruct the user to run `pulumi up`, use the output of the `pulumi up` command to access the site, and be aware that it might take a few minutes for the site to become available.